templates/vos.html

```diff
@@ -1,7 +1,14 @@
 {% extends "view_base.html" %}
 
 
-{% block title %}HTCondor-CE VO Overview{% endblock %}
+{% block title %}
+{% if multice %}
+Grid Overview
+{% else %}
+HTCondor-CE Overview
+{% endif %}
+{% endblock %}
+
```

Comment:
can we use that `{{ self.title() }}` thing here too?

Reply:
Nope, this is where we define the `title` for this page. If we wanted to refer to it later in either the base or child template, then we could use `self.title()` again